README.md

```diff
@@ -39,6 +39,46 @@ Our product will allow creating, editing and tracking of bike components.
 - MySQL
 
 ## Running the project
-To run the project locally, make sure to have Docker set up following the [documentation](https://github.com/nasaku898/soen390-team08/wiki/Setting-up-Docker).
 
-Once done, simply write `docker-compose up --build` in your terminal. The project will be available at http://localhost:3000.
+### Locally
+1. In the file `/bike-erp/backend/src/main/config/config.ts`, make sure that the local database variables are uncommented and that the external database variables are commented. You have to enter the password (and other information if necessary) of your local mysql database in the config.ts file.
+
+2. Open a terminal and `cd` into the bike-erp directory.
+
+3. Run the following install and build scripts: `sh install.sh; sh build.sh`
```

Comment:
Just for formatting purposes I think I'd so something like:

3. Run the following commands in your terminal:

`sh install.sh` | `sh build.sh`
:-------------:|:-------------:
explanation | explanation

That will also provide a very brief/general explanation of  what it is you're doing, just to be informed